src/main/java/io/github/fvarrui/javapackager/packagers/GenerateDmg.java

```diff
@@ -99,13 +100,13 @@ protected File doApply(MacPackager packager) throws Exception {
 		// mounts image
 		Logger.info("Mounting image: " + tempDmgFile.getAbsolutePath());
 		String result = execute("hdiutil", "attach", "-readwrite", "-noverify", "-noautoopen", tempDmgFile);
-		String deviceName = Arrays.asList(result.split("\n"))
-									.stream()
-									.filter(s -> s.contains(mountFolder.getAbsolutePath()))
-									.map(s -> StringUtils.normalizeSpace(s))
-									.map(s -> s.split(" ")[0])
-									.findFirst().get();
```

Comment:
This call to `get()` will throw an exception in the case that mounting an image fails.